sailthru_content/services/tests/test_sailthru_api_service.py

```diff
@@ -0,0 +1,133 @@
+#! /usr/bin/env python
```

Comment:
This line does no harm, but it's not necessary. Same applies to all other files using it.